compiler/rustc_codegen_ssa/src/mir/intrinsic.rs

```diff
@@ -555,14 +555,10 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                 return;
             }
 
-            sym::ptr_guaranteed_eq | sym::ptr_guaranteed_ne => {
+            sym::ptr_guaranteed_cmp => {
                 let a = args[0].immediate();
                 let b = args[1].immediate();
-                if name == sym::ptr_guaranteed_eq {
                     bx.icmp(IntPredicate::IntEQ, a, b)
```

Comment:
GitHub diff is making me confused.. Is this indented? It shouldn't be, right?

Reply:
it is not indented

Reply:
weird that rustfmt didn't catch this.

Reply:
it was indeed indented